app/_src/gateway/migrate-ce-to-ke.md

```diff
@@ -25,7 +25,7 @@ The following steps guide you through the migration process.
 
 1. [Download](/gateway/{{page.kong_version}}/install) the {{site.base_gateway}}
 {{page.kong_version}} Enterprise package and configure it to point to the same data store as your
-{{site.ce_product_name}} 2.8.x node. The migration command expects the data store
+{{site.ce_product_name}} {{page.kong_version}} node. The migration command expects the data store
```

Comment:
Do we even need a version number here? I don't think the version has any bearing on the instruction.

```suggestion
{{site.ce_product_name}}  node. The migration command expects the data store
```

Reply:
@lena-larionova I think that the CE node has to be the same version as the EE node you want to upgrade to, so I think that's why that's called out here. But it looks like that's already mentioned in the prerequisites section, so I think that it can be removed from this spot.

Reply:
It has to be the same, but it doesn't have to be 3.2. I think this whole topic is version-agnostic.